schemas/schema-kinds.md

```diff
@@ -143,25 +148,67 @@ valid members by one -- see the "Cardinality Examples" table below.
+losing information.
 
-The precise semantics of defaults vary per representation strategy;
+It may be interesting to note that implicits are a concept that resides in
+the *representation* clause rather than the *type* definition clause.
+This is the case because implicits do not change the cardinality of the type.
+Correspondingly, the syntatic position of an implicit declaration is on the
+end of the line declarating a field, and in paranthesis (this is the same as
+where the 'rename' and other representation-level directives can be found).
+
+The precise semantics of implicit values may vary per representation strategy;
 the discussion here is only for the general pattern, and you should also
 refer to the [reference documentation for representation strategies](./representations.md#representation-strategy-reference)
 for more details specific to the representations strategies you use.
 
-(FUTURE: there's discussion that "default" may not be the clearest name for
-this concepts.  Alternatives floated include "absent", "void", "omitted"...)
+### Combining Nullable, Optional, and Implicit
+
+The `nullable` and `optional` modifiers may be freely combined without issue,
+as may be the `nullable` and `implicit` modifiers.
+
+It is not valid to combine the `optional` and `implicit` modifiers -- since
+both regard behavior around struct fields when the data is absent in the serial
+form, it only makes sense to use exclusively one or the other on a field.
+
+There may be additional restrictions on whether `optional` and `implicit`
+modifiers may be used on a struct field based on what kind of representation
+strategy the struct has.  For example, `optional` and `implicit` cannot be
+used in fields in the middle of a struct when the representation is as tuple,
+because this could make parsing ambiguous.  The
+[reference documentation for representation strategies](./representations.md#representation-strategy-reference)
+should provide more detailed information on this.
+
+### Choosing between Optional and Implicit
+
+Use `implicit` when the absense of a value should be treated as another
+value that fits unambiguously into your domain; use `optional` when there's
+no such in-domain value.
 
-### Combining Nullable, Optional, and Default
+For example, if you have some integer field, and if `0` is distinctive from
+from the field being absent, use`optional`.
+If `0` (or some other value of your choice like `-1`) can be used as a
+entinel value in your application logic, while not being serialized as such,
+then you may wish to consider using `implicit`.
 
-The `nullable` and `default` modifiers may be freely combined without issue.
+One usage pattern to particularly note: if writing a schema for definiting
+configuration that a user may provide, and it has some concept of "defaults",
+and your application also needs to be able to *remember* whether a value was
+the default or explicitly user-provided (e.g. to re-emit the config without
+altering this)
```

Comment:
missing `.` but also maybe this thought needs to be extended slightly with advice for this situation? something like this?

> In this case, `implicit` will cause the provided value to be lost at the application-layer, whereas `optional` will expose and preserve its absence.

Reply:
Your skill with prose never ceases to humble me.  Taking that verbatim, thanks.